cmd/labs/project/labs_home.go

```diff
@@ -0,0 +1,14 @@
+package project
+
+import (
+	"context"
+	"path/filepath"
+
+	"github.com/databricks/cli/libs/env"
+)
+
+func PathInLabs(ctx context.Context, dirs ...string) string {
+	homdeDir := env.UserHomeDir(ctx)
+	prefix := []string{homdeDir, ".databricks", "labs"}
+	return filepath.Join(append(prefix, dirs...)...)
```

Comment:
Should this mkdir the path? I saw a couple calls and all of them run a mkdir.

The function could be named `EnsureLabsPathExists`.

Reply:
it's a wrapper on top of filepath.Join + env.UserHomeDir, it can't have error result and is used in many places that don't create a directory.
<img width="1628" alt="image" src="https://github.com/databricks/cli/assets/259697/375d9561-7f66-48a7-93de-481e99300d32">